README.md

```diff
@@ -26,7 +26,7 @@ The model source code, executable file and utility files are available in the [S
 
 The input datasets are available on Snellius and CRIB. First, make sure you have
 right access to the repository and data. Then, see the notebook
-[run_model_in_notebook.ipynb](https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/blob/main/notebooks/run_model_in_notebook.ipynb)
+[run_model_in_notebook.ipynb](https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/blob/main/docs/notebooks/run_model_in_notebook.ipynb)
```

Comment:
The link will show up in "Getting started" in the doc. It should point to a tag instead of main.